src/fermilib/transforms/_bksf_test.py

```diff
@@ -0,0 +1,157 @@
+                        _bksf.edge_operator_b(edge_matrix_indices, 2)))
+        self.assertTrue(qterm_b3.isclose(
+                        _bksf.edge_operator_b(edge_matrix_indices, 3)))
+
+    def test_bravyi_kitaev_fast_edgeoperator_Aij(self):
+        edge_matrix = numpy.triu(numpy.ones((4, 4)))
+        edge_matrix_indices = numpy.array(numpy.nonzero(
+                                          numpy.triu(edge_matrix) -
+                                          numpy.diag(numpy.diag(edge_matrix))))
+        correct_operators_a01 = ((0, 'X'),)
+        correct_operators_a02 = ((0, 'Z'), (1, 'X'))
+        correct_operators_a03 = ((0, 'Z'), (1, 'Z'), (2, 'X'))
+        correct_operators_a12 = ((0, 'Z'), (1, 'Z'), (3, 'X'))
+        correct_operators_a13 = ((0, 'Z'), (2, 'Z'), (3, 'Z'), (4, 'X'))
+        correct_operators_a23 = ((1, 'Z'), (2, 'Z'), (3, 'Z'),
+                                 (4, 'Z'), (5, 'X'))
+
+        qterm_a01 = QubitOperator(correct_operators_a01, 1)
+        qterm_a02 = QubitOperator(correct_operators_a02, 1)
+        qterm_a03 = QubitOperator(correct_operators_a03, 1)
+        qterm_a12 = QubitOperator(correct_operators_a12, 1)
+        qterm_a13 = QubitOperator(correct_operators_a13, 1)
+        qterm_a23 = QubitOperator(correct_operators_a23, 1)
+
+        self.assertTrue(qterm_a01.isclose(_bksf.edge_operator_aij(
+                                          edge_matrix_indices, 0, 1)))
+        self.assertTrue(qterm_a02.isclose(_bksf.edge_operator_aij(
+                                          edge_matrix_indices, 0, 2)))
+        self.assertTrue(qterm_a03.isclose(_bksf.edge_operator_aij(
+                                          edge_matrix_indices, 0, 3)))
+        self.assertTrue(qterm_a12.isclose(_bksf.edge_operator_aij(
+                                          edge_matrix_indices, 1, 2)))
+        self.assertTrue(qterm_a13.isclose(_bksf.edge_operator_aij(
+                                          edge_matrix_indices, 1, 3)))
+        self.assertTrue(qterm_a23.isclose(_bksf.edge_operator_aij(
+                                          edge_matrix_indices, 2, 3)))
+
+    def test_bravyi_kitaev_fast_jw_number_operator(self):
+        bravyi_kitaev_fast_n = _bksf.number_operator(
+                                                    self.molecular_hamiltonian)
+        jw_n = QubitOperator()
+        n_qubits = count_qubits(self.molecular_hamiltonian)
+        for i in range(n_qubits):
+            jw_n += jordan_wigner_one_body(i, i)
+        jw_eig_spec = eigenspectrum(jw_n)
+        bravyi_kitaev_fast_eig_spec = eigenspectrum(bravyi_kitaev_fast_n)
+        evensector = 0
+        for i in range(numpy.size(jw_eig_spec)):
+            if bool(numpy.size(numpy.where(jw_eig_spec[i] ==
+                                           bravyi_kitaev_fast_eig_spec))):
+                evensector += 1
+        self.assertEqual(evensector, 2**(n_qubits - 1))
+
+    def test_bravyi_kitaev_fast_jw_hamiltonian(self):
```

Comment:
It might be useful the write a comment on what exactly you are trying to compare and why ...